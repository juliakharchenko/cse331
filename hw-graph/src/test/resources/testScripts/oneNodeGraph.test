# Tests if a node can be added to the graph

# Create a graph
CreateGraph oneNodeGraph

# Add a Node
AddNode oneNodeGraph n1

# Print nodes in graph and the outgoing edges from n1
ListNodes oneNodeGraph
ListChildren oneNodeGraph n1

