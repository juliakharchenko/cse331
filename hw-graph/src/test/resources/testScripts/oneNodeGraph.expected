# Tests if a node can be added to the graph

# Create a graph
created graph oneNodeGraph

# Add a Node
added node n1 to oneNodeGraph

# Print nodes in graph and the outgoing edges from n1
oneNodeGraph contains: n1
the children of n1 in oneNodeGraph are: